lhotse/bin/modes/recipes/gigaspeech.py

```diff
@@ -12,7 +12,7 @@
 @click.argument('corpus_dir', type=click.Path(exists=True, dir_okay=True))
 @click.argument('output_dir', type=click.Path())
 @click.option('--subset', type=click.Choice(('auto',) + GIGASPEECH_PARTS), multiple=True,
-              default=['auto'], help='Which parts of Gigaspeech to download (by default XL + DEV + TEST).')
+              default=['auto'], help='Which parts of Gigaspeech to download (by default {XL} + {DEV} + {TEST}).')
```

Comment:
The curly brackets „{„ and „}” seem like an implementation detail to me — as a user I’d rather type in „XL” not „{XL}”. The GigaSpeech paper and repo also refers to these splits as „XL” rather than „{XL}”. Lhotse will correctly add the brackets before passing it into speechcolab.

Reply:
Stripping them also simplifies CLI a lot ({XL} has special meaning in bash) and avoids awkward filenames („gigaspeech_recordings_{XL}.jsonl.gz”).

Reply:
May @dophist help to explain the reason of using brackets in the subset names?